test/test.js

```diff
@@ -0,0 +1,47 @@
+import assert from "assert";
+import fs from "fs";
+
+import { unified } from "unified";
+import remarkParse from "remark-parse";
+import remarkStringify from "remark-stringify";
+import presetLintNode from "../index.js";
+import { read } from "to-vfile";
+import { reporter } from "vfile-reporter";
+
+const linter = unified()
+  .use(remarkParse)
+  .use(presetLintNode)
+  .use(remarkStringify);
+
+// Test that correctly-formatted markdown is ok.
+(async () => {
+  const file = await read(new URL("./fixtures/ok.md", import.meta.url));
+  const result = await linter.process(file);
+  assert.strictEqual(result.messages.length, 0, reporter(result));
+})().catch((e) => {
+  console.error(e);
+  process.exit(1);
+});
```

Comment:
It would be preferable to use TLA here so we can detect if a promise never settles.
```suggestion
await Promise.all([
 (async () => {
  const file = await read(new URL("./fixtures/ok.md", import.meta.url));
  const result = await linter.process(file);
  assert.strictEqual(result.messages.length, 0, reporter(result));
 })(),
 // ...
]);
```

Reply:
I originally had TLA for the test but it blew up ESLint and I didn't want to look too deeply. Future improvement?

Reply:
I suggest adding a comment explaining that. Also, we could re-use the same function for all catch handlers.

Reply:
Whoops. The real issue is that TLA is not supported in Node.js 12 and this supports Node.js 12 so that would make adding these tests a breaking change.

Reply:
> Also, we could re-use the same function for all catch handlers.

Done.

Reply:
> I suggest adding a comment explaining that.

Comment added. Optimistically marking this as resolved, but unresolve if this resolution does not resolve the issues you think need resolving. Uh, you know what I'm trying to say.